content/v2.0/backup-restore/backup.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Back up data
+seotitle: Back up data in InfluxDB
+description: >
+  Use the `influx backup` command to back up all data and metadata stored in InfluxDB.
+menu:
+  v2_0:
+    parent: Back up & restore data
+weight: 101
+related:
+  - /v2.0/backup-restore/restore/
+  - /v2.0/reference/cli/influx/backup/
+draft: true
+---
+
+Use the [`influx backup` command](/v2.0/reference/cli/influx/backup/) to back up
+all data and metadata stored in InfluxDB.
+InfluxDB copies all data and metadata to a file set stored in a specified location
+on your local filesystem.
+
+{{% warn %}}
+If your InfluxDB instance was set up using a version of InfluxDB **v2.0.0-beta.2**,
+you will not be able to successfully back up data.
+Root tokens created prior to v2.0.0-beta.2 do not have the necessary permissions.
+{{% /warn %}}
+
+{{% cloud-msg %}}
+The `influx backup` command **cannot** back up data stored in **{{< cloud-name "short" >}}**.
+{{% /cloud-msg %}}
+
+The `influx backup` command requires:
+
+- The directory path for where to store the backup file set
+- The **root authorization token** created when InfluxDB was initially set up
```

Comment:
Haven't seen the term "root authorization token" before. Do we have it defined somewhere or should we define it her?